lms/templates/admin/base.html.jinja2

```diff
@@ -3,7 +3,15 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>{% block title %}LMS Admin{% endblock %}</title>
+    <title>
```

Comment:
I have another version of this that explicitly adds a "title" block in each page.

We could go that route instead